src/pages/TeamPage/index.js

```diff
@@ -0,0 +1,349 @@
+                <Button
+                  variant="outlined"
+                  color="primary"
+                  spacing={2}
+                  container
+                  className={!isActive ? classes.team_btn : classes.team_btn1}
+                >
+                  Volunteer at Meraki
+                </Button>
+              </Grid>
+            </Stack>
+          </Container>
+          <Container
+            className={
+              !isActive
+                ? classes.team_containerTopSpace
+                : `${classes.team_responsiveContainer}`
+            }
+          >
+            <Grid container style={{ justifyContent: "center" }}>
+              <Typography
+                onClick={() => {
+                  setMembers({ volunteers: false, teamMembers: true });
+                }}
+                variant="subtitle1"
+                className={
+                  !isActive
+                    ? classes.team_selector
+                    : classes.team_MobileSelector
+                }
+                style={
+                  members.teamMembers
+                    ? {
+                        fontWeight: "bold",
+                        borderBottom: "3px solid #48a145",
+                      }
+                    : {}
+                }
+              >
+                Core Team
+              </Typography>
+              <Typography
+                onClick={() => {
+                  setMembers({ teamMembers: false, volunteers: true });
+                }}
+                variant="subtitle1"
+                className={
+                  !isActive
+                    ? classes.team_selector
+                    : classes.team_MobileSelector
+                }
+                style={
+                  members.volunteers
+                    ? {
+                        fontWeight: "bold",
+                        borderBottom: "3px solid #48a145",
+                      }
+                    : {}
+                }
+              >
+                Our Supporters
+              </Typography>
+            </Grid>
+            <Container
+              className={
+                !isActive
+                  ? `${classes.team_infoCardContaier} team-info-cards-container`
+                  : `${classes.team_infoResponsiveContainer} team-info-cards-container`
+              }
+              sx={{ marginTop: 4 }}
+            >
+              <Grid container>
+                {Object.keys(team).length ? (
+                  shuffleArray(Object.keys(team)).map((item) => {
+                    const condition = members.teamMembers
+                      ? "teamMembers"
+                      : "volunteers";
+                    if (
+                      (condition === "volunteers" &&
+                        team[item].Association === "Volunteer") ||
+                      (condition === "teamMembers" &&
+                        team[item].Association !== "Volunteer")
+                    ) {
```

Comment:
I think it might be a little clearer to write this code excerpt as something like this although the specific name of the non-volunteer association is not really used here (i.e., it could be anything that's not `Volunteer`):
```suggestion
                    const association = showTeam ? "Team" : "Volunteer";
                    if (team[item].Association === association) {
```

However, the association name `Volunteer` should probably be a constant as it's dependent on having the same name (and currently case: if value changed to `volunteer`, the above code wouldn't work) as the `Association` from the JSON file.